crates/calibration/src/center_circle/residuals.rs

```diff
@@ -0,0 +1,62 @@
+use color_eyre::Result;
+use coordinate_systems::Ground;
+use linear_algebra::Point2;
+use nalgebra::{Dyn, Owned, Vector};
+use projection::Error;
+use projection::Projection;
+use types::field_dimensions::FieldDimensions;
+
+use crate::{
+    center_circle::measurement::Measurement,
+    corrections::{get_corrected_camera_matrix, Corrections},
+    residuals::CalculateResiduals,
+};
+
+pub type Residual = Vector<f32, Dyn, ResidualStorage>;
+pub type ResidualStorage = Owned<f32, Dyn>;
+
+// TODO move above common parts
+pub struct Residuals {
+    residual_values: Vec<f32>,
```

Comment:
Could we improve the name of this field? "residual" is already expressed in the name of the struct and "values" is way too generic. What are you actually putting in here?

Reply:
Fixed; Also removed the TODOs

Renamed to `CenterCircleResiduals` and the inner field `radial_residuals`. The struct name refers to the measurement type and the inner field refers to  specifics.

Reply:
You may want to also rename the local variable in line 45... 😉 Resolve on your own if done.

Reply:
Done :D Thanks for noting it!